tutorials/physics/collision_shapes_3d.rst

```diff
@@ -0,0 +1,133 @@
+.. _doc_collision_shapes_3d:
+
+Collision shapes (3D)
+=====================
+
+Godot provides many kinds of collision shapes, with different performance and
+accuracy tradeoffs.
+
+You can define the body form of a :ref:`class_PhysicsBody` by adding one or more
+:ref:`CollisionShapes <class_CollisionShape>` as child nodes. These nodes act as
+shape holders. In these child nodes, you must add a :ref:`class_Shape`
+*resource* that will define the actual shape of the collision.
+
+.. note::
+
+    When you add multiple collision shapes to a single PhysicsBody, you don't
+    have to worry about overlapping shapes since these won't "collide" with each
+    other.
+
+Primitive collision shapes
+--------------------------
+
+Godot provides the following primitive collision shape types:
+
+- :ref:`class_BoxShape`
+- :ref:`class_SphereShape`
+- :ref:`class_CapsuleShape`
+- :ref:`class_CylinderShape` (only when using the Bullet physics engine)
+
+Using one or several primitive collision shapes, you can represent the collision
+of most smaller objects well. However, for more complex objects such as a large
+ship or a whole level, you may want to use convex shapes or concave shapes
+instead (see below).
+
+Primitive shapes are recommended for dynamic objects such as RigidBodies and
+KinematicBodies, as their behavior usually is the most reliable and predictable.
+On top of that, they often provide better performance as well.
+
+Convex collision shapes
+-----------------------
+
+:ref:`Convex collision shapes <class_ConvexPolygonShape>` are a compromise
+between primitive collision shapes and concave collision shapes. They can
+represent shapes of any complexity, but with an imporatant caveat. As their name
+implies, an individual shape can only represent a *convex* shape. For instance,
+a pyramid is *convex* but an hollow box is *concave*. To represent a concave
```

Comment:
Typo:
```suggestion
a pyramid is *convex* but a hollow box is *concave*. To represent a concave
```

Reply:
I'm not sure about this – I've found uses of "a hollow" out there.